Code/Framework/AzToolsFramework/Tests/Prefab/PrefabFocus/PrefabFocusTests.cpp

```diff
@@ -255,4 +255,37 @@ namespace UnitTest
         }
     }
 
+    TEST_F(PrefabFocusTests, VerifySelectionAfterFocus)
+    {
+        // Verify entity selection is set to nested entities and prefabs
+        AzToolsFramework::EntityIdList selectedEntities;
+
+        AzToolsFramework::ToolsApplicationRequests::Bus::BroadcastResult(
+            selectedEntities, &AzToolsFramework::ToolsApplicationRequests::GetSelectedEntities);
+        EXPECT_EQ(selectedEntities.size(), 0);
+
+        m_prefabFocusPublicInterface->FocusOnOwningPrefab(m_entityMap[Passenger1EntityName]->GetId());
+
+        AzToolsFramework::ToolsApplicationRequests::Bus::BroadcastResult(
+            selectedEntities, &AzToolsFramework::ToolsApplicationRequests::GetSelectedEntities);
+        EXPECT_EQ(selectedEntities.size(), 1);
+        EXPECT_EQ(selectedEntities[0], m_entityMap[Passenger1EntityName]->GetId());
+    }
+
+    TEST_F(PrefabFocusTests, VerifySelectionAfterFocusOnRoot)
```

Comment:
question: Is root in this context the 'level' root? Is this the thing we never want to technically select?

Reply:
Correct. We don't want to select everything when moving the focus back to the Level as that would likely result in a big selection and not provide a good UX (as it's likely not the user's intent).